src/css/tabs/firmware_flasher.less

```diff
@@ -1,4 +1,12 @@
 .tab-firmware_flasher {
+	.build-options-wrapper {
```

Comment:
Can we please just call it build-options

Reply:
technically its a wrapper around option, not the option itself